### PR TITLE
Use lighter and more neutral colors

### PR DESCRIPTION
**Pull request by Mogztter**
A picture is worth a thousand words:

result

This is a matter of taste but I think lighter colors is more neutral

### CODE REVIEW

**Comment by robotci** [comment]
Admin, can CI be enabled?

**Comment by pekkaklarck** [comment]
1. Could you attach a picture also with same data and current styles to make comparison easier?

2. Should we ask what others think on robotframework-users? Everyone who feels strongly one way or other could comment here.

**Comment by Mogztter** [comment]
hi @pekkaklarck 

> Could you attach a picture also with same data and current styles to make comparison easier?

Yes sure, I will also attach the log page for completeness :smile:

File: src/robot/htmldata/rebot/common.css [hunk 1]
```diff
@@ -174,9 +201,9 @@ th.stats-col-graph:hover {
     height: 100%;
 }
 .pass-bar {
-    background: #00F000;
+    background: #5cb85c;
 }
-/* Tablesorter - adapted from provided Blue Skin */
+/* Tablesorter - adapted from provided #428bca Skin */
```
Comment (pekkaklarck) [inline src/robot/htmldata/rebot/common.css]: Ooops. =)

**Comment by pekkaklarck** [comment]
Yes, having both original and enhanced log and report would be nice to be able to compare them. It would be good if you could add a comment enumerationg the main changes.

On a quick look this looks really good, but I need to investigate the changes a bit more. Because colors and such are so much a matter of taste, I'd like to hear general comments from others too.

**Comment by laurentbristiel** [comment]
Let's ask on the mailing list. I tend to like the new proposition. Current color are very bright/flashy. That would (IMHO) would more professional with lighter colours.

**Comment by Mogztter** [comment]
log2-old

log2-new


------
report1-old

report1-new

------

log1-old

log1-new

------

**Comment by guykisel** [comment]
Do reports no longer have the green/red background to indicate pass/fail status?

**Comment by Mogztter** [comment]
@guykisel I didn't change this behavior so the background should still be green/red

**Comment by pekkaklarck** [comment]
Sorry @Mogztter for not having time yet to really look at the changes. Next week I'm on holiday, but I'm skiing without family and actually plan to do some work in the evenings. Not sure do I actually time, though, but I promise to look at this before alpha 1. Including changes in as early preview release as possible gives us more comments and time to fine-tune changes.

Below are some comments and questions related to the PR:

1. As @guykisel noted, report background in the attached image is white. It is white also in the reference image, though. Did you print images as PDF or how did you create them? Browsers typically ignore background colors when printing by default.

2. It would be great to have the actual report and log files to look at too. You could, for example, run `./rundevel.py atest/testdata/misc` in the project root both w/ and w/o your changes. Unfortunately this issue tracker doesn't allow attaching files, but it would be great if you could upload reports and logs somewhere and link them from here. It's not that big a task for me to merge the proposed changes manually and checkout differences myself, but having examples easily available would be convenient also for others.

3. I'd like to see a list of most important changes.

4. Could you submit an issue about this? We prefer having issues about all major changes. The aforementioned list of changes could actually be part of that issue.

**Comment by Mogztter** [comment]
> As @guykisel noted, report background in the attached image is white. It is white also in the reference image, though. Did you print images as PDF or how did you create them? Browsers typically ignore background colors when printing by default.

I opened [src/robot/htmldata/rebot/report.html](https://github.com/robotframework/robotframework/blob/master/src/robot/htmldata/rebot/report.html) and [src/robot/htmldata/rebot/log.html](https://github.com/robotframework/robotframework/blob/master/src/robot/htmldata/rebot/log.html) in my browser to preview my changes. I also replaced `DeepPink` background with white in [data.js](https://github.com/robotframework/robotframework/blob/master/src/robot/htmldata/testdata/data.js#L10) to do the screenshots.

> It would be great to have the actual report and log files to look at too. You could, for example, run ./rundevel.py atest/testdata/misc in the project root both w/ and w/o your changes. Unfortunately this issue tracker doesn't allow attaching files, but it would be great if you could upload reports and logs somewhere and link them from here. It's not that big a task for me to merge the proposed changes manually and checkout differences myself, but having examples easily available would be convenient also for others.

Ok I will try to export a single html page for report.html and log.html before and after my changes.

> I'd like to see a list of most important changes.
> Could you submit an issue about this? We prefer having issues about all major changes. The aforementioned list of changes could actually be part of that issue.

Here is the related issue #1943.

**Comment by Mogztter** [comment]
Here is a live version:

http://bl.ocks.org/mogztter/raw/42540f83029a3d2139d0/log-v1.html
http://bl.ocks.org/mogztter/raw/42540f83029a3d2139d0/log-v2.html

http://bl.ocks.org/mogztter/raw/42540f83029a3d2139d0/report-v1.html
http://bl.ocks.org/mogztter/raw/42540f83029a3d2139d0/report-v2.html

**Comment by pekkaklarck** [comment]
Been sick and on holiday but now finally had time to look at the live versions. Below are some comments based on a pretty quick look. What do others think about this?

1. Lighter border color in statistics tables looks good. Definitely +1.

2. Using background color instead of text color with PASS, FAIL, WARN, etc. is an interesting idea. Change is quite big so it may take a little time for me to get used to it. +0 for now.

3. Using darker gray instead of bright yellow as a highlight color when something is activated is good. In log file when hovering over smallish buttons in suite/test/kw header the difference between original gray is quite small, though. Could the highlight gray be a little more dark? +1 for the overall change anyway.

4. Red used in the statistics table graph is too light. I agree original is flashy, but -0 for the current color.

5. Blue used as link color is a little to light too. This is especially a problem in report's statistics table. Original is perhaps too bright, but still -0 for the current color. (Blue used as link color here on GitHub actually looks nice. Not sure how it would look in log/report context.)


In general I like the new look and feel and want to have it in alpha1. I want to discuss this face-to-face with @jussimalinen when he's back from holiday next week, though.

**Comment by pekkaklarck** [comment]
@robotci: Enable.

Just realized that we can view latest logs/reports on CI once we enable it for this PR.

**Comment by robotci** [comment]
Refer to this link for build results (access rights to CI server needed): 
http://robot.radiaatto.ri.fi//job/RobotFramework-pull-requests/80/
Test FAILed.

**Comment by pekkaklarck** [comment]
Failure seems to be caused by some issue with CI server or GitHub.

**Comment by guykisel** [comment]
GitHub is getting DDoSed. http://insight-labs.org/?p=1682

**Comment by Mogztter** [comment]
Ok no problem @pekkaklarck. 

> Red used in the statistics table graph is too light. I agree original is flashy, but -0 for the current color.

I agree, the color is too soft... I will see what I can do :smile: 

> Blue used as link color is a little to light too. This is especially a problem in report's statistics table. Original is perhaps too bright, but still -0 for the current color. (Blue used as link color here on GitHub actually looks nice. Not sure how it would look in log/report context.)

Alright, I can update the stylesheet to use the link color on GitHub `#4183c4`

**Comment by pekkaklarck** [comment]
I looked at this again with @jussimalinen and we noticed that the colors look _a lot_ better on his OSX with a retina display than on my T430s with a traditional display. We then tested an external display with Jussi's OSX and colors looked more or less the same as on my machine. There was also considerable difference between how original colors looked, but there none of the real colors were clearly better or worse than others.

As a coincidence the person who provided us the external display was red-green color blind. He was able to see the difference between old red and green used in graphs, but with new colors he really couldn't tell the difference.

**Comment by zlorb** [comment]
Please see http://wistia.com/blog/heatmaps-for-colorblindness for a possible solution for red-green blindness.

**Comment by pekkaklarck** [comment]
We decided not to merge this PR directly but instead do and commit similar changes on separately. Those commits can be seen from issue #1943.

**Pull request closed without merging**